app/assets/stylesheets/custom.css.scss

```diff
@@ -135,6 +130,18 @@ h1 {
   background-color: darkgrey;
 }
 
+#flash_twitter {
+  font-family: monospace;
+  text-style: bold;
+  opacity: .8;
+  text-align: center;
+  margin-right: 60%;
+  padding:4px;
+  border-radius: 100px;
+  color: cornsilk;
+  background-color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.